termui/bug_table.go

```diff
@@ -290,10 +290,19 @@ func (bt *bugTable) render(v *gocui.View, maxX int) {
 	columnWidths := bt.getColumnWidths(maxX)
 
 	for _, excerpt := range bt.excerpts {
-		summaryTxt := fmt.Sprintf("C:%-2d L:%-2d",
-			excerpt.LenComments,
-			len(excerpt.Labels),
-		)
+		summaryTxt := fmt.Sprintf("%4d 💬", excerpt.LenComments)
```

Comment:
hmm, this will overflow if over 9999, right ? What about using `∞` after that ?

Reply:
Yes, that's right. It will be fixed the way you proposed in my next commit.